### DISCUSSION TITLE
Cannot get the output of executed command

### DISCUSSION THREAD

**Discussion by Manikantapayasam** [CrowdStrike/falconpy#759] [Q&A] [answered]
I run the below command but it only returns the status of the executed command but not the output produced by the script.

response = falconAdmin.RTR_ExecuteAdminCommand( base_command="runscript",
                                                   command_string= "runscript -CloudFile='custom_script.ps1' -CommandLine=''",
                                                   device_id=device_id,
                                                   session_id=session)

If I print the response, it is giving only the status of the command not the actual output.

Output:
{'status_code': 201, 'headers': {'Server': 'nginx', 'Date': 'Wed, 24 Aug 2022 05:47:42 GMT', 'Content-Type': 'application/json', 'Content-Length': '262', 'Connection': 
'keep-alive', 'Content-Encoding': 'gzip', 'X-Cs-Region': 'us-1', 'X-Cs-Traceid': 'xxxxx-xxx-xxx-xxxxxxxx', 'X-Ratelimit-Limit': '6000', 'X-Ratelimit-Remai: 'xxxxxx-xxxx-xxxx-xxxxxxxx'}, 'resources': [{'session_id': 'xxxxxx-xxxxx-xxxxxx-xxxxx', 'cloud_request_id': 'xxxxx-xxxx-xxxx-xxxxxxxxxxx', 'queued_command_offline': False}], 'errors': None}}

Is there anyway to get the output of executed custom script command?

**Comment by jshcodes** [comment, marked as answer]
Hi @Manikantapayasam -

Now that you've received the `cloud_request_id`, you can pass this to [`RTR_CheckAdminCommandStatus`](https://www.falconpy.io/Service-Collections/Real-Time-Response-Admin.html#rtr_checkadmincommandstatus) to retrieve the result.

[Here](https://github.com/CrowdStrike/falconpy/blob/main/samples/rtr/get_rtr_result.py) is an example utility that will look up the result for any Cloud Request ID. 

Thank you for the question!
